library/Backends/Plonk/src/Juvix/Backends/Plonk/Pipeline.hs

```diff
@@ -15,15 +15,15 @@ import qualified Juvix.Backends.Plonk.Dot as Dot
 import qualified Juvix.Backends.Plonk.Parameterization as Parameterization
 import qualified Juvix.Backends.Plonk.Types as Types
 import qualified Juvix.Core.Base.TransformExt.OnlyExts as OnlyExts
-import qualified Juvix.Core.ErasedAnn.Types as CoreErased
+import qualified Juvix.Core.Erased.Ann as ErasedAnn
+import qualified Juvix.Core.Erased.Ann.Types as ErasedAnn
```

Comment:
it looks like `Erased.Ann` reexports `Types`, so this import might be redundant?
```suggestion
```

Reply:
Updated @andy-morris